azurerm/resource_arm_automation_job_schedule_test.go

```diff
@@ -0,0 +1,217 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	uuid "github.com/satori/go.uuid"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMAutomationJobScheduleCreate(t *testing.T) {
```

Comment:
Could we create two more tests so we would have:
```
TestAccAzureRMAutomationJobSchedule_basic
TestAccAzureRMAutomationJobSchedule_complete
TestAccAzureRMAutomationJobSchedule_update
```

where the last one has 4 steps: `basic, complete, basic check_import_state`

Reply:
Let me know if this is correct.